manim/mobject/svg/style_utils.py

```diff
@@ -0,0 +1,141 @@
+"""Utility functions for parsing SVG styles."""
+
+
+__all__ = ["cascade_element_style", "parse_style", "parse_color_string"]
+
+from xml.dom.minidom import Element as MinidomElement
+from colour import web2hex
+from ...utils.color import rgb_to_hex
+
+from typing import Dict
+
+
+def cascade_element_style(
+    element: MinidomElement, inherited: Dict[str, str]
+) -> Dict[str, str]:
+    """Collect the element's style attributes based upon both its inheritance and its own attributes.
+
+    SVG uses cascading element styles. A closer ancestor's style takes precedence over a more distant ancestor's
+    style. In order to correctly calculate the styles, the attributes are passed down through the inheritance tree,
+    updating where necessary.
+
+    Note that this method only copies the values and does not parse them. See :meth:`parse_color_string` for converting
+    from SVG attributes to manim keyword arguments.
+
+    Parameters
+    ----------
+    element : :class:`MinidomElement`
+        Element of the SVG parse tree
+
+    inherited : :class:`dict`
+        Dictionary of SVG attributes inherited from the parent element.
+
+    Returns
+    -------
+        Dictionary mapping svg attributes to values with `element`'s values overriding inherited values.
```

Comment:
The return type is missing
```suggestion
    Returns
    -------
    :class:`Dict`
        Dictionary mapping svg attributes to values with `element`'s values overriding inherited values.
```

Reply:
Done in 640c50c (along with others)